packages/tracing-internal/test/browser/request.test.ts

```diff
@@ -59,363 +41,6 @@ describe('instrumentOutgoingRequests', () => {
   });
 });
 
-describe('callbacks', () => {
-  let hub: Hub;
```

Comment:
I ended up deleting these tests because they are so bound to hub/transaction/span current implementations.

We have playwright tests that cover testing these spans, those are good enough to validate that we have the expected behaviour.